doc/nginx-reverse-proxy.md

```diff
@@ -58,6 +58,7 @@ server {
 		proxy_http_version 1.1;
 		proxy_set_header Connection "";
 		proxy_set_header Forwarded "$proxy_forwarded_by;$proxy_forwarded_for;host=$http_host;proto=$scheme";
+		proxy_set_header X-nginx-request_id $request_id;
```

Comment:
In storm-tape (https://baltig.infn.it/cnafsd/storm-tape/-/blob/main/src/access_logger.cpp#L31) usiamo `x-request-id`